sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/data_transfer.py

```diff
@@ -382,39 +373,45 @@ def _customized_validate(self):
                 yaml_path="outputs.sink",
                 message="Outputs field only support one output called sink in import task",
             )
-        if "sink" in self.outputs and isinstance(self.outputs["sink"]._data, Output):
+        if (
+            "sink" in self.outputs
+            and not isinstance(self.outputs["sink"], str)
```

Comment:
This line seems unnecessary since we already check for `isinstance(Output)`. It can't be type `Output` and `str`, right? Let's remove that added line.